.github/workflows/deploy-prod.yml

```
@@ -55,139 +55,81 @@ jobs:
+              echo "✅ Elasticsearch 컨테이너 실행 중"
             else
-              echo "Elasticsearch 컨테이너가 실행 중이 아닙니다"
+              echo "❌ Elasticsearch 컨테이너 없음"
             fi
+          '
+
+      - name: 내부 헬스체크
+        run: |
+          ssh -i ~/.ssh/id_rsa ${{ secrets.EC2_USERNAME }}@${{ secrets.EC2_HOST }} '
+            echo "🏥 내부 헬스체크 시작"
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)